phoenix-core/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -565,12 +829,6 @@ private void prepareIndexMutations(
                   // add the VERIFIED cell, which is the empty cell
```

Comment:
Since the verified cell is getting added in `prepareIndexMutationsForIndexRebuild`, do you mind moving this comment inside else block or changing it?

Reply:
The location of the comment is correct. I will change the comment to:
// Add an empty column cell with the verify status